### PR TITLE
Add bitwise_or method in Calculator class

### PR DESCRIPTION
**Pull request by seungjaeryanlee**

### CODE REVIEW

File: calculator.py [hunk 1]
```diff
@@ -29,3 +29,31 @@ def div(self, a, b):
...
         """
         return a / b
+
+    def sub(self, a: float, b: float) -> float:
+        """Subtraction method for calculator class.
+
+        Note:
+            Do not include the `self` parameter in the ``Args`` section.
+        Args:
+            param1: The first parameter.
```
Comment (Curt-Park) [inline calculator.py]: Please see this example:

a (float): description

File: calculator.py [hunk 2]
```diff
@@ -29,3 +29,31 @@ def div(self, a, b):
...
+            param2: The second parameter.
+        Returns:
+            Subtraction of param1 to param2.
+
+        """
+        return a-b
```
Comment (Curt-Park) [inline calculator.py]: a - b

File: calculator.py [hunk 3]
```diff
@@ -29,3 +29,31 @@ def div(self, a, b):
...
+
+    def bitwise_or(self, a: int, b: int) -> int:
+        """Copy a bit if it exists in either operand.
+
+        Note:
+            Do not include the `self` parameter in the ``Args`` section.
+        Args:
+            param1: The first parameter.
```
Comment (Curt-Park) [inline calculator.py]: Same comment described above.

File: test_calculator.py [hunk 4]
```diff
@@ -40,3 +48,15 @@ def test_div(self):
 
         result = calc.div(1., 1.e5)
         assert result == 1.e-5
+
+    def test_bitwise_or(self):
+        """Put your test casese for bitwise_or  that starts with assert."""
+        calc = Calculator()
+        result = calc.bitwise_or(0b010000, 0b100000)
```
Comment (Curt-Park) [inline test_calculator.py]: Please make look alike how you code between 2 test functions.

File: calculator.py [hunk 5]
```diff
@@ -29,3 +29,31 @@ def div(self, a, b):
...
+        """
+        return a - b
+
+    def bitwise_or(self, a: int, b: int) -> int:
+        """Copy a bit if it exists in either operand.
+
+        Note:
+            Do not include the `self` parameter in the ``Args`` section.
+        Args:
+            param1: The left operand of bitwise or.
```
Comment (Curt-Park) [inline calculator.py]: Please fix param1, param2.

File: test_calculator.py [hunk 6]
```diff
@@ -16,8 +16,21 @@ def test_add(self):
         pass
 
     def test_sub(self):
-        """Put your test cases for sub that starts with assert."""
-        pass
+        """Check 3 test cases. Vary input signs and check the results.
```
Comment (Curt-Park) [inline test_calculator.py]: Please make just one line for the subject.

File: test_calculator.py [hunk 7]
```diff
@@ -40,3 +53,15 @@ def test_div(self):
 
         result = calc.div(1., 1.e5)
         assert result == 1.e-5
+
+    def test_bitwise_or(self):
+        """Put your test casese for bitwise_or  that starts with assert."""
```
Comment (Curt-Park) [inline test_calculator.py]: Please fix the description.

**Comment by kdh0429** [comment]
Please check reflected version of three recent comments.

**Pull request merged**